static/templates/settings/account-settings.handlebars

```diff
@@ -131,7 +131,12 @@
             <h3>{{t "User avatar" }}</h3>
 
             <div class="inline-block">
-                <img id="user-settings-avatar" src="{{ page_params.avatar_url_medium }}" />
+                <div id="user-settings-avatar">
+                    <img id="user-avatar-block" src="{{ page_params.avatar_url_medium }}" />
+                    <div id="user-avatar-source">
+                        <p> Sourced from gravatar </p>
```

Comment:
This string, like anything else in the UI, should be tagged for translation.

See zulip.readthedocs.io/en/latest/translating/index.html for how this works.